app/views/shared/views/_alerts_list.html.haml

```diff
@@ -0,0 +1,110 @@
+            %a{"href" => "#", "ng-click" => "customScope.showHostPage(item, $event)"}
+              %span.no-wrap{"tooltip" => "{{item.hostName}}",
+                            "tooltip-placement" => "bottom",
+                            "tooltip-popup-delay" => "1000"}
+                %img.list-view-type-img{"ng-src" => "{{item.hostImg}}"}
+                {{item.hostName}}
+          %div
+            %a{"href" => "#", "ng-click" => "customScope.showObjectPage(item, $event)"}
+              %span.no-wrap{"tooltip" => "{{item.objectName}}",
+                            "tooltip-placement" => "bottom",
+                            "tooltip-popup-delay" => "1000"}
+                %img.list-view-type-img{"ng-src" => "{{item.objectTypeImg}}"}
+                {{item.objectName}}
+        .col-lg-3.col-lg-push-4.col-md-4.col-md-push-3.col-sm-3.col-sm-push-3.col-xs-6.container-fluid
+          %div
+            %span.no-wrap{"tooltip" => "{{item.lastUpdate| date:'yyyy-MM-dd hh:mm:ss'}}",
+                          "tooltip-placement" => "bottom",
+                          "tooltip-popup-delay" => "1000"}
+              %span.column-label
+                = _("Updated")
+              {{item.lastUpdate| date:'yyyy-MM-dd hh:mm:ss'}}
+          %div
+            %span.no-wrap{"tooltip" => "{{item.age}}",
+                          "tooltip-placement" => "bottom",
+                          "tooltip-popup-delay" => "1000"}
+              %span.column-label
+                = _("Age")
+              {{item.age}}
+        .col-lg-4.col-lg-pull-3.col-md-3.col-md-pull-4.col-sm-3.col-sm-pull-3.col-xs-6.col
+          %span.description-column{"tooltip" => "{{item.description}}",
+                        "tooltip-placement" => "bottom",
+                        "tooltip-popup-delay" => "1000"}
+            {{item.description}}
+        .col-md-2.col-sm-2.col-xs-6.assignee-column
+          %span.ack-icon.pficon.pficon-ok{"ng-class" => "{unacknowledged: !item.acknowledged}",
+                                          "tooltip" => "{{customScope.acknowledgedTooltip}}",
+                                          "tooltip-placement" => "bottom",
+                                          "tooltip-popup-delay" => "1000"}
+          %span.assignee.no-wrap{"tooltip" => "{{item.assignee_name}}",
+                                 "tooltip-placement" => "bottom",
+                                 "tooltip-popup-delay" => "1000"}
+            {{item.assignee_name}}
+          %span.comment-icon.fa.fa-comments-o{"ng-if" => "item.numComments > 0",
+                                              "tooltip" => "{{item.commentsTooltip}}",
+                                              "tooltip-placement" => "bottom",
+                                              "tooltip-popup-delay" => "1000"}
+    %list-expanded-content
+      .row
+        .col-md-12
+          %a{"href" => "{{$parent.item.sopLink}}", "target" => "_blank", "ng-if" => "$parent.item.sopLink"}
```

Comment:
@jeff-phillips-18 Aah, so this really needs fixing on the patternfly side, `href` with `{{$parent.item.sopLink}}` still doesn't work and you need `{{$parent.$parent.item.sopLink}}` there for the `href` (and `ng-href`) to work.

Sounds like `list-expanded-content` should really be a proper component, with `item` set on its scope.